docs/sandbox.md

```diff
@@ -0,0 +1,59 @@
+# Experimental sandbox
+
+!!! warning
+    The sandbox feature is experimental and may not work as expected. The options might
+    change in future releases.
+
+Since the 0.34.0 release, `rattler-build` has a new experimental feature called
+`sandbox`. With the sandbox feature enabled (via `--sandbox`), the build process
+has much more restricted access to system resources on macOS and Linux. 
+As the sandbox feature is experimental it is disabled by default.
+
+In particular, with the default configuration, the build process can read the entire filesystem, 
```

Comment:
What was the reason again, to let it read the entire filesystem by default?

Reply:
Mostly laziness to get something off the ground. It would require some more work to figure out what needs to be avaiable in general.